spec/integration/changeable_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe Mongoid::Orderable::Mixins::Changeable do
+
```

Comment:
Rubocop is missing ;)

Reply:
Rubocop added here: https://github.com/mongoid/mongoid_orderable/pull/76